tests/certification/flow/sidecar/sidecar.go

```diff
@@ -145,6 +145,7 @@ func Stop(appID string) flow.Runnable {
 func (s Sidecar) Stop(ctx flow.Context) error {
 	var client *Client
 	if ctx.Get(s.appID, &client) {
+		client.rt.SetRunning(true)
```

Comment:
This works here but I wonder if it should go in the `Start` method, just in case it gets used for other things in the runtime in the future

Reply:
ohh, I assumed that we are already setting running flag to `true` in runtime.go's Run method, so when runtime would be invoked via cert test, then it would be set already - And, that we are just fixing some race conditions while stopping. Let me check, from your comment it seems that we are not at all setting `running` flag to true in case of cert tests.

Reply:
Correct. The running flag is set in the runtime's 'Run' method. But that method is not invoked in cert tests because the "fake runtime" does its own initialization. And that's why our tests were failing, because the running flag was unset.